galleries/users_explain/figure/interactive_guide.rst

```diff
@@ -108,24 +110,26 @@ Blocking the prompt
    backend_bases.FigureCanvasBase.stop_event_loop
 
 
-The simplest "integration" is to start the GUI event loop in
-"blocking" mode and take over the CLI.  While the GUI event loop is
-running you cannot enter new commands into the prompt (your terminal
-may echo the characters typed into the terminal, but they will not be
-sent to the Python interpreter because it is busy running the GUI
-event loop), but the figure windows will be responsive.  Once the
-event loop is stopped (leaving any still open figure windows
-non-responsive) you will be able to use the prompt again.  Re-starting
-the event loop will make any open figure responsive again (and will
-process any queued up user interaction).
+The simplest solution is to start the GUI event loop and let it run
+exclusively, which results in responsive figure windows. However, the
+CLI event loop will not run, so that you cannot enter new commands.
+We call this "blocking" mode. (your terminal may echo the typed characters,
+but they will not yet be processed by th CLI event loop because the Python
```

Comment:
```suggestion
We call this "blocking" mode. (Your terminal may echo the typed characters,
but they will not yet be processed by the CLI event loop because the Python
```